webapp/components/expression/expressionVariables.js

```diff
@@ -75,6 +77,20 @@ const getChildDefVariables = ({ survey, nodeDefContext, nodeDefCurrent, mode, la
   return variables
 }
 
+export const getParentsInOrder = ({ parent, survey, includeMultiples = false }) => {
```

Comment:
Method to get the list of the nodes in the survey which has children and are not multiple in order, 
This method is used later to get the list of uuid and the index of each parent

Reply:
what do you mean this method is used later?

Reply:
the response of this method is used later*

Reply:
what does this method do?